known-limitations.md

```diff
@@ -64,28 +64,28 @@ As a workaround, [execute the file from the command line](use-the-built-in-sql-c
 
 When executing an [`ALTER TABLE ADD COLUMN`](add-column.html) statement with a [`DEFAULT`](default-value.html) expression, new values generated:
 
-- use the default [search path](sql-name-resolution.html#search-path) regardless of the search path configured in the current session via `SET SEARCH_PATH`.
-- use the UTC time zone regardless of the time zone configured in the current session via [`SET TIME ZONE`](set-time-zone.html).
-- have no default database regardless of the default database configured in the current session via [`SET DATABASE`](set-database.html).
-- use the transaction timestamp for the `statement_timestamp()` function regardless of the time at which the `ALTER` statement was issued.
+- Use the default [search path](sql-name-resolution.html#search-path) regardless of the search path configured in the current session via `SET SEARCH_PATH`.
```

Comment:
I'd made these lower-case because each is a completion of the sentence started before the colon. I usually like to introduce a list with a complete clause but didn't have the time here. If you can find a way to do that, it'd be better. Otherwise, I'm fine with this change.